### ISSUE TITLE
Prevent selecting or uploading wrong image types in Image editor

### ISSUE THREAD

**Issue by drlimbo** [neos/neos-ui#2176]
### Description
 I can choose non-image files (PDF) in Media Browser and get an exception

### Steps to Reproduce

1. Press the Media Button near the Image
2. Choose a non-image-file (PDF)
3. You get an Error 💥

**Example YAML**

```
teaserImage:
  type: 'Neos\Media\Domain\Model\ImageInterface'
  ui:
    label: 'Teaser Image'
    inspector:
      group: 'settings'
      editorOptions:
        accept: 'image/png'
```

#### Expected behavior
I can choose only image-files in Media Browser

#### Actual behavior
 I can choose non-image files (PDF) in Media Browser and get an exception

![mediabrowser](https://user-images.githubusercontent.com/1118783/47008303-5938c080-d13a-11e8-9bea-8f5321c407b1.png)

### Affected Versions
Neos: 4.1.7
Neos-UI: v1.3.4

**Comment by markusguenther** [comment]
It turns out that this issue is not related to the react-ui. It also did not work in the old UI and the real problem occurs in the media browser.

We can close the issue and I created a new one in the media browser.
https://github.com/neos/neos-development-collection/issues/2255

**Comment by markusguenther** [comment]
Issue has been resolved for the neos 4.0 branch and will be fixed when the up merges take place.
